### ISSUE TITLE
Lando fails to restart consistenly after successful start

### ISSUE THREAD

**Issue by RichardDavies** [lando/lando#1538]
**Tell us about your setup**

v3.0.0-rc.13 on Windows 10 Professional (I recently upgraded from beta 44)

**Tell us about your `.lando.yml`**

```yml
name: portlandor
recipe: pantheon
config:
  framework: drupal8
  site: portlandor
  id: 5c6715db-abac-4633-ada8-1c9efe354629
  # Disable the SOLR index
  index: false
services:
  appserver:
    build_as_root:
      - /bin/sh -c "cp /app/cop_ca_*.crt /usr/local/share/ca-certificates/"
      - /bin/sh -c "update-ca-certificates"
    run_as_root:
      - /bin/sh -c "if [ ! -d /var/www/.terminus/plugins/terminus-secrets-plugin ]; then { mkdir -p /var/www/.terminus/plugins; composer create-project -d /var/www/.terminus/plugins pantheon-systems/terminus-secrets-plugin:~1; chown -R www-data:www-data /var/www/.terminus; } fi"
    xdebug: true
    config:
      conf: xdebug.ini
tooling:
  drush:
    service: appserver
    cmd: drush --root=/app/web
```

`pantheon.yml`

```yml
# Put overrides to your pantheon.upstream.yml file here.
# For more information, see: https://pantheon.io/docs/pantheon-yml/
api_version: 1

web_docroot: true

php_version: 7.2

# Add a comment to trigger a deploy on Pantheon
workflows:
  clear_cache:
    after:
      - type: webphp
        description: Post deployment notification to Slack
        script: private/scripts/slack_deploy_notification.php
  clone_database:
    after:
      - type: webphp
        description: Post deployment notification to Slack
        script: private/scripts/slack_deploy_notification.php
  create_cloud_development_environment:
    after:
      - type: webphp
        description: Post deployment notification to Slack
        script: private/scripts/slack_deploy_notification.php
  deploy:
    after:
      - type: webphp
        description: Import configuration from .yml files
        script: private/scripts/drush_post_deploy.php
      - type: webphp
        description: Post deployment notification to Slack
        script: private/scripts/slack_deploy_notification.php
  sync_code:
    after:
      - type: webphp
        description: Post deployment notification to Slack
        script: private/scripts/slack_deploy_notification.php
```

**Tell us about the command you were running**

```bash
# I installed v3.0.0-rc.13 on top of beta 44 and ran:
lando destroy -y
lando poweroff
docker system prune -f
docker image prune -f
docker volume prune -f
# Then I updated my .lando.yml as above and
# I deleted my .lando directory, restarted Docker, and
# Start Lando from a clean slate
lando start
# The project builds successfully without any errors
# Verify site loads as expected. Everything looks fine up to this point.
# Now lets restart just for fun...
lando restart
# It restarts just fine, let's do it again...
lando restart
# It restarts just fine again, Now the "fun" begins...
lando restart --verbose
# It DOESN'T correctly restart this time or a 4th time
# and I literally haven't done anything except restart it multiple times
# In fact, this also stalls in exactly the same spot:
lando rebuild -y
```

**Tell us about the error you got**

On both of the failed restart attempts and the failed rebuild attempt, it gets stalled after this output:

```
info: Checking to see if https://localhost:32893 is ready.
info: Checking to see if http://localhost:32894 is ready.
info: Checking to see if http://localhost:32897 is ready.
info: Checking to see if http://portlandor.lndo.site is ready.
info: Checking to see if https://portlandor.lndo.site is ready.
info: Checking to see if https://localhost:32896 is ready.
```

Here's the tail of my `lando.log` file after a failed restart:

```
{"level":"verbose","message":"Healthcheck for portlandor = {\"service\":\"edge_ssl\",\"health\":\"healthy\"}","timestamp":"2019-03-15T19:14:07.820Z"}
{"level":"verbose","message":"Healthcheck for portlandor = {\"service\":\"edge\",\"health\":\"healthy\"}","timestamp":"2019-03-15T19:14:07.820Z"}
{"level":"verbose","message":"Healthcheck for portlandor = {\"service\":\"appserver_nginx\",\"health\":\"healthy\"}","timestamp":"2019-03-15T19:14:07.820Z"}
{"level":"verbose","message":"Healthcheck for portlandor = {\"service\":\"cache\",\"health\":\"healthy\"}","timestamp":"2019-03-15T19:14:07.820Z"}
{"level":"verbose","message":"Healthcheck for portlandor = {\"service\":\"database\",\"health\":\"healthy\"}","timestamp":"2019-03-15T19:14:07.820Z"}
{"level":"verbose","message":"Healthcheck for portlandor = {\"service\":\"appserver\",\"health\":\"healthy\"}","timestamp":"2019-03-15T19:14:07.820Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:07.820Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:07.821Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:07.821Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:07.821Z"}
{"level":"debug","message":"Emitting event pre-engine-list","timestamp":"2019-03-15T19:14:07.821Z"}
{"level":"debug","message":"Event pre-engine-list has 0 listeners","timestamp":"2019-03-15T19:14:07.821Z"}
{"level":"debug","message":"Emitting event post-engine-list","timestamp":"2019-03-15T19:14:07.834Z"}
{"level":"debug","message":"Event post-engine-list has 0 listeners","timestamp":"2019-03-15T19:14:07.834Z"}
{"level":"info","message":"Connected portlandor_appserver_1 to the landonet","timestamp":"2019-03-15T19:14:11.546Z"}
{"level":"info","message":"Connected portlandor_edge_1 to the landonet","timestamp":"2019-03-15T19:14:13.049Z"}
{"level":"info","message":"Connected portlandor_cache_1 to the landonet","timestamp":"2019-03-15T19:14:13.278Z"}
{"level":"info","message":"Connected portlandor_edge_ssl_1 to the landonet","timestamp":"2019-03-15T19:14:13.519Z"}
{"level":"info","message":"Connected portlandor_database_1 to the landonet","timestamp":"2019-03-15T19:14:13.891Z"}
{"level":"info","message":"Connected portlandor_appserver_nginx_1 to the landonet","timestamp":"2019-03-15T19:14:14.092Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:14.092Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.092Z"}
{"level":"debug","message":"Engine is up.","timestamp":"2019-03-15T19:14:14.359Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:14.359Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.359Z"}
{"level":"debug","message":"Emitting event pre-engine-list","timestamp":"2019-03-15T19:14:14.359Z"}
{"level":"debug","message":"Event pre-engine-list has 0 listeners","timestamp":"2019-03-15T19:14:14.359Z"}
{"level":"debug","message":"Emitting event post-engine-list","timestamp":"2019-03-15T19:14:14.370Z"}
{"level":"debug","message":"Event post-engine-list has 0 listeners","timestamp":"2019-03-15T19:14:14.370Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:14.370Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.370Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:14.370Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.370Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:14.370Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.371Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:14.371Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.371Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:14.371Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.371Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:14.371Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.371Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:14.374Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.374Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:14.374Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.374Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:14.375Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.375Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:14.375Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.375Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:14.375Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.375Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:14.375Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:14.375Z"}
{"level":"debug","message":"Emitting event pre-engine-isRunning","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Event pre-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Emitting event pre-engine-isRunning","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Event pre-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Emitting event pre-engine-isRunning","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Event pre-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Emitting event pre-engine-isRunning","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Event pre-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Emitting event pre-engine-isRunning","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Event pre-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Emitting event pre-engine-isRunning","timestamp":"2019-03-15T19:14:14.376Z"}
{"level":"debug","message":"Event pre-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:14.377Z"}
{"level":"debug","message":"Emitting event post-engine-isRunning","timestamp":"2019-03-15T19:14:16.313Z"}
{"level":"debug","message":"Event post-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:16.313Z"}
{"level":"debug","message":"Emitting event post-engine-isRunning","timestamp":"2019-03-15T19:14:18.539Z"}
{"level":"debug","message":"Event post-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:18.539Z"}
{"level":"debug","message":"Emitting event post-engine-isRunning","timestamp":"2019-03-15T19:14:19.302Z"}
{"level":"debug","message":"Event post-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:19.302Z"}
{"level":"debug","message":"Emitting event post-engine-isRunning","timestamp":"2019-03-15T19:14:20.116Z"}
{"level":"debug","message":"Event post-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:20.116Z"}
{"level":"debug","message":"Emitting event post-engine-isRunning","timestamp":"2019-03-15T19:14:20.649Z"}
{"level":"debug","message":"Event post-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:20.649Z"}
{"level":"debug","message":"Emitting event post-engine-isRunning","timestamp":"2019-03-15T19:14:20.953Z"}
{"level":"debug","message":"Event post-engine-isRunning has 0 listeners","timestamp":"2019-03-15T19:14:20.953Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:20.953Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:20.953Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:20.953Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:20.953Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:20.954Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:20.954Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:20.954Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:20.954Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:20.954Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:20.954Z"}
{"level":"debug","message":"Emitting event pre-engine-up","timestamp":"2019-03-15T19:14:20.954Z"}
{"level":"debug","message":"Event pre-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:20.954Z"}
{"level":"debug","message":"Engine is up.","timestamp":"2019-03-15T19:14:21.305Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:21.305Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:21.305Z"}
{"level":"debug","message":"Emitting event pre-engine-scan","timestamp":"2019-03-15T19:14:21.305Z"}
{"level":"debug","message":"Event pre-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.305Z"}
{"level":"debug","message":"Engine is up.","timestamp":"2019-03-15T19:14:21.313Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:21.313Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:21.313Z"}
{"level":"debug","message":"Emitting event pre-engine-scan","timestamp":"2019-03-15T19:14:21.313Z"}
{"level":"debug","message":"Event pre-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.313Z"}
{"level":"debug","message":"Engine is up.","timestamp":"2019-03-15T19:14:21.314Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:21.314Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:21.314Z"}
{"level":"debug","message":"Emitting event pre-engine-scan","timestamp":"2019-03-15T19:14:21.315Z"}
{"level":"debug","message":"Event pre-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.315Z"}
{"level":"debug","message":"Engine is up.","timestamp":"2019-03-15T19:14:21.321Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:21.321Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:21.321Z"}
{"level":"debug","message":"Emitting event pre-engine-scan","timestamp":"2019-03-15T19:14:21.321Z"}
{"level":"debug","message":"Event pre-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.321Z"}
{"level":"debug","message":"Emitting event post-engine-scan","timestamp":"2019-03-15T19:14:21.340Z"}
{"level":"debug","message":"Event post-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.340Z"}
{"level":"debug","message":"Emitting event post-engine-scan","timestamp":"2019-03-15T19:14:21.342Z"}
{"level":"debug","message":"Event post-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.342Z"}
{"level":"debug","message":"Emitting event post-engine-scan","timestamp":"2019-03-15T19:14:21.342Z"}
{"level":"debug","message":"Event post-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.342Z"}
{"level":"debug","message":"Emitting event post-engine-scan","timestamp":"2019-03-15T19:14:21.343Z"}
{"level":"debug","message":"Event post-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.343Z"}
{"level":"debug","message":"Engine is up.","timestamp":"2019-03-15T19:14:21.346Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:21.347Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:21.347Z"}
{"level":"debug","message":"Emitting event pre-engine-scan","timestamp":"2019-03-15T19:14:21.347Z"}
{"level":"debug","message":"Event pre-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.347Z"}
{"level":"debug","message":"Engine is up.","timestamp":"2019-03-15T19:14:21.348Z"}
{"level":"debug","message":"Emitting event post-engine-up","timestamp":"2019-03-15T19:14:21.348Z"}
{"level":"debug","message":"Event post-engine-up has 0 listeners","timestamp":"2019-03-15T19:14:21.348Z"}
{"level":"debug","message":"Emitting event pre-engine-scan","timestamp":"2019-03-15T19:14:21.349Z"}
{"level":"debug","message":"Event pre-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.349Z"}
{"level":"debug","message":"Emitting event post-engine-scan","timestamp":"2019-03-15T19:14:21.358Z"}
{"level":"debug","message":"Event post-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.358Z"}
{"level":"debug","message":"Emitting event post-engine-scan","timestamp":"2019-03-15T19:14:21.392Z"}
{"level":"debug","message":"Event post-engine-scan has 0 listeners","timestamp":"2019-03-15T19:14:21.392Z"}
{"max":16,"waitCodes":[400,502,404],"level":"debug","message":"Starting url scan with opts","timestamp":"2019-03-15T19:14:21.393Z"}
{"level":"info","message":"Checking to see if https://localhost:32893 is ready.","timestamp":"2019-03-15T19:14:21.394Z"}
{"level":"info","message":"Checking to see if http://localhost:32894 is ready.","timestamp":"2019-03-15T19:14:21.395Z"}
{"level":"info","message":"Checking to see if http://localhost:32897 is ready.","timestamp":"2019-03-15T19:14:21.395Z"}
{"level":"info","message":"Checking to see if http://portlandor.lndo.site is ready.","timestamp":"2019-03-15T19:14:21.396Z"}
{"level":"info","message":"Checking to see if https://portlandor.lndo.site is ready.","timestamp":"2019-03-15T19:14:21.396Z"}
{"level":"info","message":"Checking to see if https://localhost:32896 is ready.","timestamp":"2019-03-15T19:14:21.397Z"}
```

**Tell us generally about your bug**

After Lando successfully starts from a clean slate, it seems to have trouble consistently restarting or rebuilding successfully. It gets stalled toward to end of the process while checking to see if the services are ready. It will just wait there for hours until you CTRL-C or otherwise kill the process. 

Once it fails to restart successfully, the only way I've been able to fix it is to destroy the app, poweroff Lando, prune my Docker containers, restart Docker, and then it will successfully start once (maybe 2-3 times if you're lucky) before it starts stalling again on restart/reload.

This behavior has been confirmed on another coworker's PC as well (also Win10Pro).

**Tell us more**

After aborting the stalled restart or rebuild by using CTRL-C, `lando list` reports:

```bash
$ lando list
{
  portlandor: [
    {
      service: 'edge_ssl',
      name: 'portlandor_edge_ssl_1',
      app: 'portlandor',
      src: [
        'C:\\Users\\csrichardd\\Sites\\portlandor\\.lando.yml'
      ],
      kind: 'app',
      status: 'Up 3 minutes'
    },
    {
      service: 'edge',
      name: 'portlandor_edge_1',
      app: 'portlandor',
      src: [
        'C:\\Users\\csrichardd\\Sites\\portlandor\\.lando.yml'
      ],
      kind: 'app',
      status: 'Up 3 minutes'
    },
    {
      service: 'appserver_nginx',
      name: 'portlandor_appserver_nginx_1',
      app: 'portlandor',
      src: [
        'C:\\Users\\csrichardd\\Sites\\portlandor\\.lando.yml'
      ],
      kind: 'app',
      status: 'Up 3 minutes'
    },
    {
      service: 'cache',
      name: 'portlandor_cache_1',
      app: 'portlandor',
      src: [
        'C:\\Users\\csrichardd\\Sites\\portlandor\\.lando.yml'
      ],
      kind: 'app',
      status: 'Up 3 minutes'
    },
    {
      service: 'database',
      name: 'portlandor_database_1',
      app: 'portlandor',
      src: [
        'C:\\Users\\csrichardd\\Sites\\portlandor\\.lando.yml'
      ],
      kind: 'app',
      status: 'Up 3 minutes (healthy)'
    },
    {
      service: 'appserver',
      name: 'portlandor_appserver_1',
      app: 'portlandor',
      src: [
        'C:\\Users\\csrichardd\\Sites\\portlandor\\.lando.yml'
      ],
      kind: 'app',
      status: 'Up 3 minutes'
    }
  ],
  _global_: [
    {
      service: 'proxy',
      name: 'landoproxyhyperion5000gandalfedition_proxy_1',
      app: '_global_',
      src: 'unknown',
      kind: 'service',
      status: 'Up About an hour'
    }
  ]
}
```

**Comment by RichardDavies** [comment]
We've been able to resolve this issue by completely uninstalling Lando and Docker, then reinstalling. It would be helpful to emphasize the need to uninstall prior to updating in https://docs.devwithlando.io/guides/updating-to-rc2.html

**Comment by RichardDavies** [comment]
This problem has suddenly resurfaced. I've been (successfully) running v3.0.0-rc.14 for many weeks and then suddenly started having this same problem again when starting Lando. I'm mystified as to why it fails to start because I don't see any errors/issues in the log file (as far as I can tell)...

Does anyone have any ideas what's going on? How can I resolve this? I could probably fix it by completely uninstalling Docker and Lando again and then re-installing, but that doesn't seem like an acceptable solution. Surely there's a less drastic solution to this... please help!

**Comment by RichardDavies** [comment]
I'm still trying to troubleshoot this issue and I can consistently reproduce it on multiple PCs. I'm starting to think it's an underlying problem with Docker (or maybe the landoproxyhyperion5000gandalfedition_proxy_1). Here's some additional details I've discovered.

It seems to work fine with a fresh install of our project. I can start Lando and access the site (and get the Drupal install screen). I can restart Lando multiple times and it continues to work fine.

But as soon as I import our database that's when it starts to change. I run `lando latest` and `lando refresh` which are defined as:

```
  latest:
    description: Download and import the latest database from Dev
    cmd:
      - appserver: echo "Downloading latest database..."
      - appserver: rm -f artifacts/database.tar.gz
      - appserver: terminus backup:get portlandor.dev --element=database --to=artifacts/database.tar.gz
      - appserver: echo "Importing database..."
      - database: /helpers/sql-import.sh artifacts/database.tar.gz
      - appserver: echo "Clearing cache..."
      - appserver: drush cr -y
  refresh:
    description: Refresh branch by installing Composer dependencies and importing Drupal config, etc. Run after every merge from master.
    cmd:
      - appserver: echo "Installing Composer dependencies..."
      - appserver: composer install
      - appserver: echo "Clearing cache..."
      - appserver: drush cr -y
      - appserver: echo "Run database updates..."
      - appserver: drush updb -y
      - appserver: echo "Importing Drupal config..."
      - appserver: drush cim -y
      - appserver: echo "Running cron..."
      - appserver: drush core:cron -y
      - appserver: echo "Clearing cache..."
      - appserver: drush cr -y
      - appserver: echo "Building theme asset files..."
      - node: npm -C web/themes/custom/cloudy run build:dev
```

And I can then access our site in it's current state. But now if I do a `lando restart  -verbose` it will hang on one of the "info: Checking to see if [some URL] is ready" lines. The site will no longer come up in the browser.

If I execute `curl portlandor.lndo.site` on the host it just hangs as well. However, if I ssh into one of the containers and then run `curl portlandor.lndo.site` it returns the HTML of our homepage. 

So basically the site seems to be working okay within the container, but from the host it's not accessible. Which leads me to believe it's a problem with either Docker or the Lando proxy container (that seems to be triggered by a database import). But I'm not very knowledgeable about either of those two items to troubleshoot further. Any ideas, @pirog (or anyone)?

**Comment by RichardDavies** [comment]
For what it's worth, we seem to have found a workaround to this issue by switching Docker to the Edge channel and updating to Docker 2.0.4.1.
